applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_solver_vmsmonolithic.py

```diff
@@ -208,4 +199,6 @@ def Initialize(self):
         self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DYNAMIC_TAU, self.settings["dynamic_tau"].GetDouble())
         self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.OSS_SWITCH, self.settings["oss_switch"].GetInt())
 
-        print ("Monolithic MPI solver initialization finished.")
+        if (KratosMPI.mpi.rank == 0):
```

Comment:
you could use what I did in StructuralMechanics: `print_on_rank_zero`
I think this is more convenient until the logger works in MPI